src/pages/AgreementPage/AgreementPage.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import ico_roundcheck_filled from "../../assets/images/24x24/ico_roundcheck_filled.svg";
+import ico_roundcheck_outline from "../../assets/images/24x24/ico_roundcheck_outline.svg";
+
+const AgreementPage: React.FC = () => {
+    const [isAllChecked, setIsAllChecked] = useState(false);
+    const [isAgeChecked, setIsAgeChecked] = useState(false);
+    const [isAdChecked, setIsAdChecked] = useState(false);
+
+    const handleAllCheck = () => {
+        const newCheckState = !isAllChecked;
+        setIsAllChecked(newCheckState);
+        setIsAgeChecked(newCheckState);
+        setIsAdChecked(newCheckState);
+    };
+
+    const handleAgeCheck = () => {
+        const newCheckState = !isAgeChecked;
+        setIsAgeChecked(newCheckState);
+        setIsAllChecked(newCheckState && isAdChecked);
+    }
+
+    const handleAdCheck = () => {
+        const newCheckState = !isAdChecked;
+        setIsAdChecked(newCheckState);
+        setIsAllChecked(newCheckState && isAgeChecked);
+    }
+
+    const isFormValid = isAgeChecked;
+
+    return (
+        <div className="w-[390px] mx-auto pt-[72px] bg-[#F8F9FC]">
+            <h1 className="text-2xl font-pretendard font-bold ml-6 mb-[64px]">
+                <span className="text-point500">회원가입</span>을 위한<br />약관에 동의해주세요.
```

Comment:
```suggestion
                <span className="text-point500">회원가입</span>을 위한<p>약관에 동의해주세요.</p>
```
p 태그로 변경해주면 좋을 것 같아요 !! :)